src/main/java/lotto/model/TicketMachine.java

```diff
@@ -0,0 +1,36 @@
+package lotto.model;
+
+import java.util.List;
+
+public class TicketMachine {
+
+    private static final long COST_PER_TICKET = 1000L;
```

Comment:
```suggestion
    private static final long COST_PER_TICKET = 1_000L;
```
자바 7부터 숫자에 언더바(_)를 사용할 수 있습니다.
좀 더 가독성있게 표현할 수 있으니 참고 하시기 바랍니다.